### ISSUE TITLE
Ability to change default recipe location for new users. Default folder should be hidden? 

### ISSUE THREAD

**Issue by hobbsecky** [nextcloud/cookbook#666]
On my nextcloud all new registered users are assigned to "users" group.
I created a group share folder ./Recipes (I prefer this folder to be hidden so as not to clutter the file manager view)
All "users" have read, write and admin has read, write, delete priviledge on ./Recipes
This gives the ability for users to have one common cookbook.

It would be nice to have the ability to change the default folder for new users.

Thanks!

**Comment by hobbsecky** [comment]
This is issue 666. PS....I am not Satan.

**Comment by davidwozabal** [comment]
I have the identical problem on a NextCloud installation. 

How can I configure the cookbook such that newly added users share one folder for their recipes? Ideally this should work without any required input from the users and without manually re-setting the folder for every new user in the respective cookbook of that user.

Thanks!

**Comment by christianlupus** [comment]
The problem with this issue is: how should this work exactly?

We currently have a translated name for the cookbook folder. That means that the name of the folder to be created depends on the local language. Am English speaking person will have a folder `Recipes` while a German speaking person will have a folder `Rezepte` etc.

How should this feature request be implemented in this logic? Do you mean to have a checkbox that allows to automatically create the folder avoiding to the language seeing? Is this, what you are suggesting?

**Comment by davidwozabal** [comment]
No, my suggestion would be to leave the default setting as it is (with automatically created folders that are only accessible to the specific user and whose names respect the language settings) but to add an option to configure the cookbook in such a way that there is one large shared cookbook for everyone in a folder that is created by the admin (using whichever location the admin sees fit). Hence, no folder would have to be created automatically for that but the admin would rather choose one (!) pre-existing folder for that purpose.

That is the way, I am currently using the cookbook. The problem, however, is that I have to manually change the recipe folder name for every user, which is a slight hassle.
